modules/conversantBidAdapter.js

```diff
@@ -176,7 +176,18 @@ export const spec = {
    * @return {Bid[]} An array of bids which were nested inside the server.
    */
   interpretResponse: function(serverResponse, bidRequest) {
-    return converter.fromORTB({request: bidRequest.data, response: serverResponse.body});
+    const ortbBids = converter.fromORTB({request: bidRequest.data, response: serverResponse.body});
+    const ortbAuctionConfigs = deepAccess(serverResponse, 'body.ext.igi') || [];
```

Comment:
is this not handled here now? 

https://github.com/prebid/Prebid.js/commit/0ee4efdfde777045bdf5e00539efcfb878ff22d6#diff-24ee01f9467dc2a1704da10ca938150df2b3386d6b5b732870760dbc7cd1c4d7R373

cc @dgirardi

Reply:
looks like as of today it is.  I'll go back through my changes and check it out

Reply:
yes - this is no longer necessary. note that if you decide to keep it, `paapiAuctionConfigs` was renamed to just `paapi` (since it can also contain `igs` now) .